docs-sphinx/ak.behavior.rst

```diff
@@ -309,6 +310,74 @@ Similarly for overriding ``abs``
 
 and all other ufuncs.
 
+Mixin decorators
+================
+The pattern of adding additional properties and function overrides to records
+and arrays of records is quite common, and can be nicely described by the "mixin"
+idiom: a class with no constructor that is mixed with both the :doc:`_auto/ak.Array` and :doc:`_auto/ak.Record`
+class as to create new derived classes. The :doc:`_auto/ak.behaviors.mixins.mixin_class` and :doc:`_auto/ak.behaviors.mixins.mixin_class_method`
+python decorators assist with some of this boilerplate. Consider the ``Point`` class
+from above; we can implement all the functionality so far described as follows:
+
+.. code-block:: python
+    @ak.mixin_class(ak.behavior)
```

Comment:
I think the missing space here makes this not render in the sphinx docs

Reply:
@jpivarski

Reply:
That's true. (One of the quirks of ReST.) I didn't check the look of the documentation before merging it, but I can put in a correction. (They're updated with each push to master, not each new version deployed.)

Reply:
That's true. (One of the quirks of ReST.) I didn't check the look of the documentation before merging it, but I can put in a correction. (They're updated with each push to master, not each new version deployed.)

Reply:
Fixed in #360.